src/sentry/tasks/statistical_detectors.py

```diff
@@ -79,7 +79,7 @@ def detect_transaction_trends(project_ids: List[int], **kwargs) -> None:
 
 @instrumented_task(
     name="sentry.tasks.statistical_detectors.detect_function_trends",
-    queue="performance.statistical_detector",
+    queue="profiling.statistical_detector",
```

Comment:
Normally changing queues is not a safe operation to do like this as there may be tasks left in the queue but we're at such low throughput that this is not an issue.